app/serializers/api/v2/form_response_serializer.rb

```diff
@@ -13,6 +13,10 @@ class FormResponseSerializer < ActiveModel::Serializer
 
       include TimestampableModel
 
+      def form_field_values
+        object.form_field_values.where(latest: true)
```

Comment:
you can simply write object.form_field_values.latest
Rails automatically adds scopes for boolean columns